docs/usage/types.md

```diff
@@ -904,6 +908,15 @@ The following arguments are available when using the `conbytes` type function
 - `max_length: int = None`: maximum length of the byte string
 - `strict: bool = False`: controls type coercion
 
+### Arguments to `condate`
+The following arguments are available when using the `condate` type function
+
+- `strict_formats: List[str] = None`: list of datetime.datetime.strpformat style date formats that will be accepted
```

Comment:
Let's remove this, it's not supported in V2 so it'll be confusing if we add it now in V1.

(the bounds constraints are already implemented in V2).